vl-test.js

```diff
@@ -0,0 +1,42 @@
+import { css, html, LitElement } from 'https://unpkg.com/lit-element@latest/lit-element.js?module';
+import {VlElement, define} from '/node_modules/vl-ui-core/vl-core.js';
+import './vl-modal.src.js';
+import '/node_modules/vl-ui-button/vl-button.js';
+import '/node_modules/vl-ui-input-field/vl-input-field.js';
+
+export class VlTest extends LitElement {
```

Comment:
Op zich is dit wel nuttig om er bij te hebben.
De naam vl-test is echter wat vaag. Beter LitElementDemo of iets dergelijk. Ik zou de locatie van het bestand ook verplaatsen naar de demo folder, omdat die enkel daar gebruikt wordt.